ts/Accessibility/Accessibility.ts

```diff
@@ -254,9 +254,13 @@ class Accessibility {
         this.keyboardNavigation.update(kbdNavOrder);
 
         // Handle high contrast mode
+        // Should only be applied once, and not if explicitly disabled
         if (
-            !chart.highContrastModeActive && // Only do this once
-            whcm.isHighContrastModeActive()
+            !chart.highContrastModeActive &&
+            !a11yOptions.highContrastMode === false && (
```

Comment:
`a11yOptions.highContrastMode !== false`? (save 1 byte 🥳)

Reply:
🥳 
![image](https://github.com/highcharts/highcharts/assets/7478772/66ada162-9a3c-4e56-82dc-a9509753d264)

Reply:
wait a minute...

Reply:
Gzip works in mysterious ways..